decred_wallet/Features/Security/SecurityViewController.swift

```diff
@@ -61,11 +61,17 @@ class SecurityViewController: SecurityBaseViewController {
         tabController?.selectedIndex = 0
         btnPassword.backgroundColor = #colorLiteral(red: 0.9529411765, green: 0.9607843137, blue: 0.9647058824, alpha: 1)
         btnPin.backgroundColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
+//        btnPin.layer.borderWidth = 2.0 // Add border on activation
```

Comment:
If these commented lines of code are not necessary, take them off.

Reply:
this was for https://github.com/raedahgroup/dcrios/projects/1#card-21587731
i await clearance to complete the fix